packages/format-library/src/text-color/inline.js

```diff
@@ -145,7 +137,11 @@ function ColorPicker( { name, property, value, onChange } ) {
 	return (
 		<ColorPalette
 			value={ activeColors[ property ] }
-			onChange={ onColorChange }
+			onChange={ ( color ) => {
+				onChange(
+					setColors( value, name, colors, { [ property ]: color } )
+				);
+			} }
```

Comment:
I checked other usages of the `ColorPalette` component; memoization isn't required.

Reply:
For clarity and my own edification, would pleased elaborate on what you mean by "required?" 

My interpretation is that you mean that memoizing this callback is unnecessary as `ColorPalette` itself it not memoized. Is that accurate?

Thanks! 🙇🏻

Reply:
>My interpretation is that you mean that memoizing this callback is unnecessary as ColorPalette itself it not memoized. Is that accurate?

Correct and the `onChange` prop isn't a dependency for any side-effect ( `useEffect` ).